web/wp-config.php

```diff
@@ -36,7 +36,7 @@
 /**
  * Force SSL
  */
-define( 'FORCE_SSL_ADMIN', true );
```

Comment:
Did you mean to check in this change?  It not, can you revert please?

Reply:
@cjyabraham we removed the changes we made.